pypiserver/_app.py

```diff
@@ -81,6 +81,11 @@ def log_error(http_error):
     log.info(config.log_err_frmt, vars(http_error))
 
 
+@app.route("/health")
+def health():
+    return 'Ok.'
```

Comment:
Dot seems to be unnecessary:

```suggestion
    return 'Ok'
```

Reply:
That's a good idea, indeed, the dot definitely can be skipped 👍  :)